### DISCUSSION TITLE
Security/Cryptography/Digital-Signature

### DISCUSSION THREAD

**Discussion by giscus** [xy-241/CS-Notes#44] [General]
# Security/Cryptography/Digital-Signature

Abstract The purpose of digital signature is to prove that something from one party is really from that party, that thing is unchanged from the moment it leaves that party to us receiving ...

https://notes.yxy.ninja/Security/Cryptography/Digital-Signature

**Comment by FrancisRalph** [comment]
damnyum, amazing stuff!

**Comment by xy-241** [comment]
d
